projects/storefrontlib/src/cms-components/checkout/components/checkout-progress/multiline-titles.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'cxMultiLine',
+})
+export class MultiLinePipe implements PipeTransform {
```

Comment:
👍 

but need the unit test and the travis build is failing because it can't find this pipe, and